azurerm/data_source_express_route_circuit.go

```diff
@@ -0,0 +1,204 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-08-01/network"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmExpressRouteCircuit() *schema.Resource {
```

Comment:
could we add [some documentation for this data source](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/website/docs/d) and a [link to the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L71-L94)?